src/encoded/static/components/summary.js

```diff
@@ -353,16 +353,11 @@ class SummaryBody extends React.Component {
         this.context.navigate(href);
     }
     render() {
-        let clearButton;
         const searchQuery = url.parse(this.props.context['@id']).search;
         const query = new QueryString(searchQuery);
-        if (query) {
-            // If we have a 'type' query string term along with others terms, we need a Clear Filters
-            // button.
-            const nonPersistentQuery = query.clone();
-            nonPersistentQuery.deleteKeyValue('type');
-            clearButton = nonPersistentQuery.queryCount() > 0 && query.queryCount('type') > 0;
-        }
+        const nonPersistentQuery = query.clone();
+        nonPersistentQuery.deleteKeyValue('?type');
+        const clearButton = nonPersistentQuery.queryCount() > 0 && query.queryCount('?type') > 0;
```

Comment:
Oh! Yeah that works too. I forgot the way I usually do this is to use url.parse(this.props.context['@id']).query instead of .search. They give you the same thing, but .query doesn’t have the question mark. I wonder if QueryString should check for a leading question mark and ignore it, so you can use either?

Reply:
That could be handy! Or, I could remember to use `.query` instead.